src/test/java/resources/validateTxn.feature

```diff
@@ -1,18 +1,18 @@
 
- Feature: Validate Txn Data
+Feature: Validate Txn Data
 
- Scenario: VT-001 GSMA Transfer Api Payer Invalid Test
-    Given I can create GSMATransferDTO with invalid payer information
-    And I have tenant as "paymentbb2"
-    When I call the GSMATransfer endpoint with expected status of 200
-    And I should be able to parse transactionId from response
-    Then I should poll the transfer query endpoint with transactionId until status is populated for the transactionId
+   Scenario: VT-001 GSMA Transfer Api Payer Invalid Test
+      Given I can create GSMATransferDTO with invalid payer information
```

Comment:
Do not include format changes in your PR. you don't want to take ownership for the files that you did not edit purposefully.

Reply:
ok will keep that in mind